arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/precompiles/ModexpMetadata.java

```diff
@@ -135,17 +135,19 @@ public boolean extractExponent() {
 
   public Bytes base() {
     Bytes unpadded = Bytes.EMPTY;
-    if (callData.size() >= BASE_MIN_OFFSET) {
-      final int sizeToExtract = Math.min(bbsInt(), callData.size() - MBS_MIN_OFFSET);
-      unpadded = callData.slice(MBS_MIN_OFFSET, sizeToExtract);
+    final int firstOffset = BASE_MIN_OFFSET;
+    if (callData.size() > firstOffset) {
+      final int sizeToExtract = Math.min(bbsInt(), callData.size() - firstOffset);
+      unpadded = callData.slice(BASE_MIN_OFFSET, sizeToExtract);
```

Comment:
We were using `MBS_MIN_OFFSET` as opposed to `BASE_MIN_OFFSET`. Also the comparisons have to be strict.